### PR TITLE
try to add a simple github action

### PR DESCRIPTION
**Pull request by cfbolz**

### CODE REVIEW

**Comment by mattip** [comment]
cool. Of course this is not really portable since it builds on `ubuntu-20.04`. Is it worth building inside a `manylinux_2014` docker? The artifact does have value for bisecting failures like the one in #4816.

File: .github/workflows/translate.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Translate PyPy
+
+on:
+  push:
+    branches: [ "main", "py3.9", "py3.10"]
+  pull_request:
+    branches: [ "main", "py3.9", "py3.10"]
```
Comment (mattip) [inline .github/workflows/translate.yml:7]: This [will add](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) the ability to kick of this action via a button on the workflow page.

```suggestion
    branches: [ "main", "py3.9", "py3.10"]
  workflow_dispatch:    
```

**Comment by cfbolz** [comment]
@mattip yeah, it's not properly portable at all, but without `--make-portable` it doesn't even work on my ubuntu 23.10 laptop, so this is still an improvement. I would like to merge this as it is, we can find out how to do Docker in a github action as a later step. ok for you?

**Comment by mattip** [comment]
Yes, makes sense.

**Comment by cfbolz** [comment]
cool, merging then :blush:

**Pull request merged**